### DISCUSSION TITLE
Module to render Markdown

### DISCUSSION THREAD

**Discussion by markdav-is** [oqtane/oqtane.framework#678] [General]
I use markdown rendering components in the place of wysiwyg components for rending user supplied content.  The limitations help keep the site style consistent.   I think my first public module will be a markdown module.  unless it already exists.

**Comment by mikecasas** [comment]
I haven't seen a markdown module.

**Comment by sbwalker** [comment]
I have used https://github.com/lunet-io/markdig in the past for integrating markdown into applications - you might want to give it a try.

**Reply by markdav-is** [reply]
markdig is great,  I use that for my markdown razor component.  Now I need to find a good markdown editor to add to the module...

**Reply by hishamco** [reply]
> Now I need to find a good markdown editor to add to the module.

In Orchard Core we're using [SimpleMDE](https://github.com/sparksuite/simplemde-markdown-editor) which is fit our needs
